#! /bin/sh /usr/share/dpatch/dpatch-run
## 20-use-debian.init-script.dpatch by Mario Izquierdo (mariodebian) <mariodebian@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use Debian init.d script

@DPATCH@
diff -urNad p910nd-0.92~/Makefile p910nd-0.92/Makefile
--- p910nd-0.92~/Makefile	2009-02-08 18:06:47.000000000 +0100
+++ p910nd-0.92/Makefile	2009-02-08 18:15:56.000000000 +0100
@@ -19,7 +19,7 @@
 
 PROG = p910nd
 CONFIG = p910nd.conf
-INITSCRIPT = p910nd.init
+INITSCRIPT = debian/p910nd.init
 MANPAGE = p910nd.8
 INSTALL = install
 BINDIR = /usr/sbin
